website/src/components/ToolPreview.vue

```diff
@@ -1,21 +1,15 @@
 <template>
 	<b-link class="ToolPreview" :href="tool.link">
-		<b-card
-			class="ToolPreview__card"
-			:title="tool.name"
-			:img-src="tool.image"
-			:img-alt="tool.name"
-			img-top
-			text-variant="dark"
-			bg-variant="info">
-			<b-card-text class="ToolPreview__card__text" v-text="tool.description"/>
-			<section class="ToolPreview__card__tags">
-				<b-badge
+		<div class="ToolPreview__card">
+			<img v-bind:src="tool.image" class="ToolPreview__card__image"/>
```

Comment:
if we remove the `src` property from this image and instead simply have a span here we can put some css magic on the class `ToolPreview__card__image`;

```
<!-- html -->
<span :style="{ 'background-image': tool.image } class="ToolPreview__card__image"/>
...

// scss

&__image {
    width: 100%;
    height: 300px; // or something that looks right
    background-size: cover;
    background-repeat: no-repeat;
    background-position: center;
}
```

Reply:
I've created a PR around this comment into your branch here:
https://github.com/holistic-web/toolbox/pull/120